public/styles.css

```diff
@@ -384,12 +374,16 @@ section#pageThree {
 /* #chartBoxInside {
     height: 1000px;
 } */
+.toolBarWrapper {
+    width: 100%;
```

Comment:
Any reason the toolbar-wrapper is be at 100% width?

Reply:
I do think 100% is good to have and easier to control since we are doing a responsive page...

Reply:
Deleted it. No use of it. Forgot to delete it after playing around

Reply:
Gtocha. I also didn't realize at the time that the `toolBarWrapper` was wrapping the entire dashboard not just the toolbar.